Cabal/src/Distribution/PackageDescription/Check/Warning.hs

```diff
@@ -1086,6 +1090,8 @@ ppExplanation (COptONumber prefix label) =
     ++ " --disable-optimization flag."
 ppExplanation (COptCPP opt) =
   "'cpp-options: " ++ opt ++ "' is not a portable C-preprocessor flag."
+ppExplanation (OptJSP opt) =
+  "'jsp-options: " ++ opt ++ "' is not a portable C-preprocessor flag."
```

Comment:
Should it be "not a portable **JS**-preprocessor flag"?

Reply:
I'd prefer "JavaScript" instead of JS

Reply:
Done